.github/workflows/windows.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         shell: bash -l {0}
         run: |
           conda info
-          pip install --requirement requirements-full.txt
+          pip install --requirement requirements-full.txt --user
```

Comment:
--user breaks virtual environment. With --user pip will install packages globally that will create a massive mess (speaking from experience).

Reply:
This was me trying to see what was going on. I'll close this now